src/components/HOCs/WithTaskBundle/WithTaskBundle.js

```diff
@@ -42,6 +41,20 @@ export function WithTaskBundle(WrappedComponent) {
       }
     }
 
+    removeTaskFromBundle = async (bundleId, taskId) => {
+      this.setState({loading: true})
+      if(this.state.taskBundle.taskIds.length == 2) {
```

Comment:
this should be '==='
https://www.freecodecamp.org/news/loose-vs-strict-equality-in-javascript/#:~:text=The%20%3D%3D%20operator%20performs%20a,well%20as%20the%20same%20value).